### ISSUE TITLE
Android Offline Signing via QR code

### ISSUE THREAD

**Issue by Scollop355** [spesmilo/electrum#8561]
Hi, I can't find my old posts to update you on this problem so had to open a new post.

Electrum on Android.

Single signature wallets signing off line via qr code working on 4.3.4 but not functioning on
versions 4.4.4 and 4.4.5 give following error.

"Warning: Some data (prev txs / "full utxos") was left out of the QR code as it would not fit.  This might cause issues if signing offline.  As a workaround, try exporting the tx as file or text instead."

I have several wallets which all worked for offline signing without the error until the new GUI versions of Electrum were released.

**Comment by SomberNight** [comment]
please don't open duplicate issues. (https://github.com/spesmilo/electrum/issues/8538)

----

It is not an error but a warning. IIRC the only thing changed between e.g. 4.3.4 and 4.4.5 in this respect is that we now show the warning. The underlying logic is still the same, and the text of the warning also applied to older versions, it was just not shown.

Have you actually tested offline signing? If it worked on older versions, it should still work.

**Comment by Scollop355** [comment]
Yes, I did test 4.4.5 with offline signing and it did work despite the
warning, but I only did that because there was only a small amount of funds
on the wallet.

Thanks for the clarification that this issue was in older versions but not
displayed.  Now that I know this, I will avoid offline signing via QR code
and stick to the copy and paste method to be safe.

**Comment by Scollop355** [comment]
Sorry.  I don't understand what you mean when you state "Would it make
sense to remove it from the QR code window, and to display it only at
signing time, if the signing device actually requires the previous utxos in
order to sign?"
